test/edu/wright/cs/raiderplanner/model/RoomTest.java

```diff
@@ -0,0 +1,99 @@
+/*
```

Comment:
This header comment is not properly formatted.  Please have a look at other examples from the project and match those.  Make sure to include your name.

Reply:
I believe this is corrected now.